package.json

```diff
@@ -27,5 +27,8 @@
     "@azure/static-web-apps-cli": "^0.8.3",
     "azure-functions-core-tools": "^3.0.4484",
     "concurrently": "^7.1.0"
+  },
```

Comment:
Add the Dependency to the APP level package.json instead of root level package.json.